hasher-matcher-actioner/hmalib/lambdas/hashing.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import functools
+import boto3
+import os
+import datetime
+import json
+
+from mypy_boto3_dynamodb import DynamoDBServiceResource
+from mypy_boto3_sqs import SQSClient
+
+from threatexchange.content_type.meta import get_content_type_for_name
+from threatexchange.content_type.photo import PhotoContent
+from threatexchange.content_type.video import VideoContent
+from threatexchange.signal_type.md5 import VideoMD5Signal
+from threatexchange.signal_type.pdq import PdqSignal
+
+from hmalib.common.logging import get_logger
+from hmalib import metrics
+from hmalib.common.message_models import URLSubmissionMessage
+from hmalib.hashing.unified_hasher import UnifiedHasher
+from hmalib.models import PipelineHashRecord
+from hmalib.common.content_sources import URLContentSource
+
+logger = get_logger(__name__)
+sqs_client = boto3.client("sqs")
```

Comment:
rm

Reply:
Good spot! We use this in a later commit.